.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,27 +7,24 @@ assignees: ''
 
 ---
 
-**Describe the bug**
+## Description
 A clear and concise description of what the bug is.
 
-**To Reproduce**
-Steps to reproduce the behavior:
+##  Steps To Reproduce
```

Comment:
The only thing I can suggest is perhaps moving this below ## Expected Behavior

Reply:
Like that idea. Take a look at [this](https://github.com/Thenlie/Streamability/issues/195) issue. I made the steps heading bold, but not an actual heading. I thought that also looked nice.